package.json

```diff
@@ -54,40 +54,39 @@
         "@types/js-yaml": "^4.0.9",
         "@types/luxon": "^3.6.2",
         "@types/mocha": "^10.0.10",
-        "@types/node": "^24.0.4",
+        "@types/node": "^24.0.14",
         "@types/nunjucks": "3.2.6",
         "@types/supertest": "^6.0.3",
         "@types/uuid": "10.0.0",
-        "@typescript-eslint/eslint-plugin": "^8.35.0",
-        "@typescript-eslint/parser": "^8.35.0",
+        "@typescript-eslint/eslint-plugin": "^8.37.0",
+        "@typescript-eslint/parser": "^8.37.0",
         "cheerio": "^1.1.0",
-        "eslint": "^9.29.0",
+        "eslint": "^9.31.0",
         "eslint-plugin-check-file": "^3.3.0",
         "eslint-plugin-import": "^2.32.0",
-        "eslint-plugin-n": "^17.20.0",
+        "eslint-plugin-n": "^17.21.0",
         "eslint-plugin-node": "11.1.0",
         "eslint-plugin-promise": "^7.2.1",
         "eslint-plugin-unused-imports": "^4.1.4",
-        "globals": "^16.2.0",
+        "globals": "^16.3.0",
         "gulp": "^5.0.1",
         "gulp-clean": "^0.4.0",
         "gulp-concat": "^2.6.1",
         "gulp-sass": "^6.0.1",
         "gulp-uglify": "^3.0.2",
         "husky": "^9.1.7",
-        "jest": "^30.0.3",
+        "jest": "^30.0.4",
         "mocha": "^11.7.1",
         "nock": "14.0.5",
         "node-mocks-http": "^1.17.2",
         "nodemon": "^3.1.10",
         "nyc": "^17.1.0",
         "sass": "1.89.2",
-        "sonarqube-scanner": "3.5.0",
-        "supertest": "^7.1.1",
+        "supertest": "^7.1.3",
         "ts-jest": "^29.4.0",
         "ts-node": "^10.9.2",
         "typescript": "^5.8.3",
-        "typescript-eslint": "^8.35.0"
+        "typescript-eslint": "^8.37.0"
     },
     "@comments devDependencies": {
         "@package sonarqube-scanner": [
```

Comment:
so do we still want this sonar-scanner comment here?

Reply:
Good spot, i'll remove it.